css/styles.css

```diff
@@ -1,52 +1,57 @@
-* {
+*, *::before, *::after {

```

Comment:
Ja bym IE8 się nie przejmowała. Coraz mniej jest aplikacji które wspiera to 💩 My tu wspieramy najnowsze przeglądarki.